Zend/tests/011.phpt

```diff
@@ -64,19 +76,12 @@ bool(true)
 bool(true)
 bool(false)
 bool(false)
-
-Warning: First parameter must either be an object or the name of an existing class in %s on line %d
-NULL
-
-Warning: First parameter must either be an object or the name of an existing class in %s on line %d
-NULL
-
-Warning: First parameter must either be an object or the name of an existing class in %s on line %d
-NULL
+property_exists(): Argument #1 ($object_or_class) must be of type object|string
```

Comment:
For a bonus point, I think the ", %s given" part could be added as well.

Reply:
I'm not sure we can, how would an object be converted to the ``%s`` format?

Reply:
generally, we use `zend_zval_type_name(zval)` for this, but I haven't looked at what's going on with `property_exists`

Reply:
Yeah, we should add the `, %s given` here, to keep it consistent with other TypeErrors.

Reply:
Okay I'm dumb, I was working too much on the encoding warning to error promotion and thought you were asking to display the string instead of the type in the ``%s given`` *sight*

Reply:
Don't worry :) I also made some dumb errors yesterday (didn't have enough time).